landbosse/model/CollectionCost.py

```diff
@@ -17,7 +17,7 @@
 import traceback
 import pandas as pd
 
-from .SharedOutputMixin import SharedOutputMixin
+from .CostModule import CostModule
```

Comment:
You will see this change in several files because I changed the name of `SharedOutputMixin` to `CostModule`